pkg/operator/apiserver/audit/audit_policies.go

```diff
@@ -8,6 +8,7 @@ import (
 
 	corev1 "k8s.io/api/core/v1"
 	kyaml "k8s.io/apimachinery/pkg/util/yaml"
+	auditconfig "k8s.io/apiserver/pkg/apis/audit/v1"
 
```

Comment:
@sttts I just wanted to call out that this import is needed for the output of the new `GetAuditPolicy` func. This apiserver was not imported before.